test/e2e_node/docker_test.go

```diff
@@ -70,4 +78,170 @@ var _ = framework.KubeDescribe("Docker features [Feature:Docker]", func() {
 			}
 		})
 	})
+
+	Context("when live-restore is enabled [Serial] [Slow] [Disruptive]", func() {
+		It("containers should not be disrupted when the daemon is down", func() {
+			const (
+				podName           = "live-restore-test-pod"
+				containerName     = "live-restore-test-container"
+				volumeName        = "live-restore-test-volume"
+				timestampFilename = "timestamp"
+			)
+
+			isSupported, err := isDockerLiveRestoreSupported()
+			framework.ExpectNoError(err)
+			if !isSupported {
+				framework.Skipf("Docker live-restore is not supported.")
+			}
+
+			By("Check whether live-restore is enabled.")
+			isEnabled, err := isDockerLiveRestoreEnabled()
+			framework.ExpectNoError(err)
+			if !isEnabled {
+				framework.Skipf("Docker live-restore is not enabled.")
+			}
+
+			// Creates a temporary directory that will be mounted into the
+			// container, serving as the communication channel between the host
+			// and the container.
+			By("Create temporary directory for mount.")
+			tempDir, err := ioutil.TempDir("", "")
+			framework.ExpectNoError(err)
+			defer func() {
+				By("Remove temporary directory.")
+				os.RemoveAll(tempDir)
+			}()
+
+			// Creates a container that writes the current timestamp every
+			// second to the timestamp file. We will be able to tell whether
+			// the container is running by checking if the timestamp increases.
```

Comment:
Offline discussed. The network should not be disrupted when docker is down. When test locally with `make test-e2e-node`, because kubenet is disabled, it's possible that network is disrupted. However, when running `make test-e2e-node REMOTE=true`, kubenet is enabled, network should not be disrupted ideally.

We'll test with kubenet again to see whether network is affected.